index.bs

```diff
@@ -333,6 +333,8 @@ An attribution report is a [=struct=] with the following items:
 :: A point in time.
 : <dfn>source identifier</dfn>
 :: An opaque string.
+: <dfn>delivered</dfn>
+:: A boolean (default false).
```

Comment:
Is there a benefit to specifying a default here instead of explicitly initializing this to `false` when the value is constructed?